lib/generators/suspenders/styles_generator.rb

```diff
@@ -39,6 +40,25 @@ def configure_application_stylesheet
           TEXT
         end
       end
+
+      def install_postcss_url
+        run "yarn add postcss-url"
+      end
+
+      def configures_postcss
+        File.delete(postcss_config) if File.exist?(postcss_config)
```

Comment:
Unfortunately, `copy_file` does not override existing files. So, we need to delete it first.